package.json

```diff
@@ -123,7 +127,7 @@
       "text",
       "text-summary"
     ],
-    "all": true,
-    "check-coverage": true
+    "report-dir": "test/coverage",
```

Comment:
could we please move this under coverage dir rather than on `test/coverage`?

Reply:
Sure will do